tests/trainer/data_flow/test_train_loop_flow_scalar.py

```diff
@@ -330,3 +318,47 @@ def training_step(self, batch, batch_idx):
         if not batch_idx % 2:
             assert out.training_step_output_for_epoch_end == [[]]
         assert out.signal == 0
+
+
+def collate_none_when_even():
```

Comment:
nit: can you move this function inside the `TestModel` class in the test?

Reply:
Done :smiley: